scripts/eurostat/health_determinants/alcohol_consumption/process.py

```diff
@@ -0,0 +1,820 @@
+        'ED3-4':
+        'UpperSecondaryEducationOrPostSecondaryNonTertiaryEducation',
+        'ED3_4':
+            'UpperSecondaryEducationOrPostSecondaryNonTertiaryEducation',
+        'ED5_6' : 'TertiaryEducationStageOneOrTertiaryEducationStageTwo',
+        'ED5-8': 'TertiaryEducation',
+        'ED5_8': 'TertiaryEducation',
+        'TOTAL': 'Total'
+        }})
+    return df
+
+def _replace_quant_inc(df:pd.DataFrame) -> pd.DataFrame:
+    """
+    Replaces values of a single column into true values
+    from metadata returns the DF.
+    Args: 
+        df (pd.DataFrame): df as the input, to change column values
+    Returns:
+        df (pd.DataFrame): modified df as output
+    """
+    df = df.replace({'quant_inc': {
+        'TOTAL': 'Total',
+        'QU1': 'IncomeOf0To20Percentile',
+        'QU2': 'IncomeOf20To40Percentile',
+        'QU3': 'IncomeOf40To60Percentile',
+        'QU4': 'IncomeOf60To80Percentile',
+        'QU5': 'IncomeOf80To100Percentile'
+        }})
+    return df
+
+def _replace_deg_urb(df:pd.DataFrame) -> pd.DataFrame:
+    """
+    Replaces values of a single column into true values
+    from metadata returns the DF
+    Args: 
+        df (pd.DataFrame): df as the input, to change column values
+    Returns:
+        df (pd.DataFrame): modified df as output
+    """
+    df = df.replace({'deg_urb': {
+        'TOTAL':'Total',
+        'DEG1':'Urban',
+        'DEG2':'SemiUrban',
+        'DEG3':'Rural',
+        }})
+    return df
+
+def _replace_c_birth(df:pd.DataFrame) -> pd.DataFrame:
+    """
+    Replaces values of a single column into true values
+    from metadata returns the DF
+    Args: 
+        df (pd.DataFrame): df as the input, to change column values
+    Returns:
+        df (pd.DataFrame): modified df as output
+    """
+    df = df.replace({'c_birth': {
+        'EU28_FOR': 'ForeignBornWithinEU28',
+        'NEU28_FOR': 'ForeignBornOutsideEU28',
+        'FOR': 'ForeignBorn',
+        'NAT': 'Native'
+    }})
+    return df
+
+def _replace_citizen(df:pd.DataFrame) -> pd.DataFrame:
+    """
+    Replaces values of a single column into true values
+    from metadata returns the DF
+    Args: 
+        df (pd.DataFrame): df as the input, to change column values
+    Returns:
+        df (pd.DataFrame): modified df as output
+    """
+    df = df.replace({'citizen': {
+        'EU28_FOR': 'WithinEU28AndNotACitizen',
+        'NEU28_FOR': 'CitizenOutsideEU28',
+        'FOR': 'NotACitizen',
+        'NAT': 'Citizen'
+    }})
+    return df
+
+class EuroStatAlcoholConsumption:
+    """
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+    def __init__(self, input_files: list, csv_file_path: str,\
+        mcf_file_path: str, tmcf_file_path: str) -> None:
+        self.input_files = input_files
+        self.cleaned_csv_file_path = csv_file_path
+        self.mcf_file_path = mcf_file_path
+        self.tmcf_file_path = tmcf_file_path
```

Comment:
Can you please change scope to internal / protected.

Reply:
Done the changes.